build2/KoreBuild.ps1

```diff
@@ -0,0 +1,104 @@
+Write-Host -ForegroundColor Green "Starting KoreBuild 2.0 ..."
+
+$KoreBuildScriptDir = Convert-Path $PSScriptRoot
+$env:REPO_FOLDER = Convert-Path (Get-Location)
+$env:KOREBUILD_FOLDER = Split-Path -Parent $KoreBuildScriptDir
+
+$RID = "win7-x64";
+if($env:PROCESSOR_ARCHITECTURE -eq "x86") {
+    $RID = "win7-x86";
+}
+
+$ToolsDirectory = $env:KOREBUILD_TOOLS_DIRECTORY
+if(!$ToolsDirectory) {
+    $ToolsDirectory = Join-Path $KoreBuildScriptDir "Tools"
+}
+
+$MSBuildProjDir = Join-Path $ToolsDirectory "MSBuildTools"
+$MSBuildDir = Join-Path $MSBuildProjDir "publish"
+$KoreBuildTasksPath = Join-Path $ToolsDirectory "KoreBuild"
+
+$KoreBuildLog = Join-Path $KoreBuildScriptDir "..\korebuild.log"
```

Comment:
nit: save it to a log folder and put a timestamp on it? i find it usefully to save logs for comparison to detect issues. - non block suggestion.